subprojects/language-native/src/testFixtures/groovy/org/gradle/language/cpp/AbstractCppComponentIntegrationTest.groovy

```diff
@@ -145,11 +138,7 @@ abstract class AbstractCppComponentIntegrationTest extends AbstractNativeLanguag
 
     @Override
     protected String getDefaultArchitecture() {
-        return supports32BitArchitectureOnly() ?  "x86" : super.defaultArchitecture
-    }
-
-    protected String getCurrentHostArchitectureDsl() {
-        return supports32BitArchitectureOnly() ? ".x86" : ""
+        return super.defaultArchitecture
```

Comment:
❓ I think we can remove this method completely as it seems it only call the super method unless it's a property named `defaultArchitecture`.

Reply:
Yep, just missed it - removed.